pkgs/development/libraries/clutter-gtk/default.nix

```diff
@@ -14,6 +14,12 @@ stdenv.mkDerivation rec {
     sha256 = "01ibniy4ich0fgpam53q252idm7f4fn5xg5qvizcfww90gn9652j";
   };
 
+  postPatch = ''
```

Comment:
For GNOME packages, we are standardizing the attribute ordering to https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/attribute-ordering.md.

Reply:
Done. Thanks!